### PR TITLE
Project Business Site - By Linnéa Helén

### PR DESCRIPTION
**Pull request by Polkapetunian**
I went for the blue goals and one red (putting icons on the accordion)

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,30 +1,156 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
+    <link rel="stylesheet" href="./style.css ">
```
Comment (EmOlsz) [inline code/index.html:6]: Thank you for remembering about `./` in path to file :)

File: code/index.html [hunk 2]
```diff
@@ -1,30 +1,156 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@500&family=Parisienne&display=swap" rel="stylesheet">
+    <title>Buttercream Piping Masterclass</title>
     <!-- dont forget to add a css file and link it here! -->
   </head>
   <body>
+  
+    
     <!-- video or image as a header is cool :) -->
+    <header class="header">
+      <picture>
```
Comment (EmOlsz) [inline code/index.html:17]: Well done hero image!

File: code/index.html [hunk 3]
```diff
@@ -1,30 +1,156 @@
...
+
+    </header>
     <!-- Signup form -->
 
+    
+
+    <form class="form" action="http://httpbin.org/anything" method="POST">
+      <h3>Please choose your date!</h3>
+
+      <br>
```
Comment (EmOlsz) [inline code/index.html:41]: We shouldn't use `<br />` tags, they are deprecated. Their main features is that they break the line, but you don't need it over here, `<h3 />` tag is doing that anyway. Let's delete this `<br />`

File: code/index.html [hunk 4]
```diff
@@ -1,30 +1,156 @@
...
+
+      <label class="radio radio1">
+        <input type="radio" name="date" value="Sat 10th Apr 2021"> 
+        &nbsp;&nbsp;Sat 10th Apr 2021
+      </label>
+
+      <label class="radio radio2">
+        <input type="radio" name="date" value="Sat 17th Apr 2021">
```
Comment (EmOlsz) [inline code/index.html:49]: Well done connection between `<label />` and `<input />`!

File: code/index.html [hunk 5]
```diff
@@ -1,30 +1,156 @@
...
     </section>
-    <!-- dont forget to link the javascript file here! -->
+      <!-- dont forget to link the javascript file here! -->
+      <script src="./script.js"></script>
+
+    <footer class="footer">
+      <p class="copyright">&#169 2021 Linnéa Helén</p>
+      <p> <span>Photo by <a href="https://unsplash.com/@mmw189?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">
+        M. W</a> on <a href="https://unsplash.com/collections/71561283/cupcake?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">
+          Unsplash</a></span></p>
```
Comment (EmOlsz) [inline code/index.html:151]: Ups, very weird code formatting. Let's fix this

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,238 @@
...
+  font-family:'Noto Sans JP', sans-serif;
+  color: #660033;
+  border: 1px solid #660033; 
+  width: 100px;
+}
+
+.accordion {
+  flex-direction: column;
+  width: 250px; 
+  margin: 15px 0 15px 0;
```
Comment (EmOlsz) [inline code/style.css:92]: We can simplify this property :

```
margin: 15px 0;
```

File: code/style.css [hunk 7]
```diff
@@ -1,7 +1,238 @@
...
+.active .plus {
+  display: none;
+}
+
+.active .minus {
+  display: block;
+}
+
+.answer {
+  height: 0px;
```
Comment (EmOlsz) [inline code/style.css:129]: No need to specify unit if value is equal 0 :

```
height: 0;
```

File: code/style.css [hunk 8]
```diff
@@ -1,7 +1,238 @@
...
+  overflow:hidden;
+  transition: height .3s ease;
+  padding: 0 10px 0 10px;
```
Comment (EmOlsz) [inline code/style.css:132]:
```
padding: 0 10px;
```

File: code/style.css [hunk 9]
```diff
@@ -1,7 +1,238 @@
...
+font-size: 40px;
+}
+
+.ornament {
+  width: 220px; 
+}
+
+.form {
+width: 400px; 
+margin: 60px 0 30px 0;
```
Comment (EmOlsz) [inline code/style.css:156]: Ups, wrong indentations

File: code/style.css [hunk 10]
```diff
@@ -1,7 +1,238 @@
...
+@media (min-width: 1200px) {
+
+body {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 100vh 90vh 10vh;
+  grid-template-areas: 
+  "header header"
+  "form accordion"
+  "footer footer";
```
Comment (EmOlsz) [inline code/style.css:190]: Hmm, I think you don't need this ;)

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Hello Linnéa! Very nice project, great job :)
Some minor things needs to be fixed though, take a look at my comments for more details.
Moreover, feel free to check out [Week 3 Master document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475)

**Comment by EmOlsz** [comment]
Thank you for fixing `<br />` tags Linnea :) I will be waiting for other changes, after seeing them I will accept whole project!

**Comment by Polkapetunian** [comment]
I have fixed the remaining issues. =)

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**